<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setTempSecretDialog</class>
 <widget class="QDialog" name="setTempSecretDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="Label">
         <property name="text">
          <string>原密码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="old_password_LineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Label_4">
         <property name="text">
          <string>授权密码</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="temp_password_LineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Label_5">
         <property name="text">
          <string>确认授权密码</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="temp_password_again_LineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Label_6">
         <property name="text">
          <string>授权截止日</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="valid_time_dateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="submit_pushButton">
       <property name="text">
        <string>提交更改</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
